src/js/configuration_panel/panel-events.js

```diff
@@ -97,7 +104,7 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
      */
     gamepad.configurationPanel.handleEvents.changeConfigMenuOptions = function (that, dropdownMenu) {
         var selectedAction = $(dropdownMenu).val(),
-            dropdownClassName = dropdownMenu.classList[0],
+            dropdownClassName = dropdownMenu.classList[1],
```

Comment:
If someone makes even small changes to your markup (such as adding their own class for style purposes), it will throw this off.  This can probably wait and be fixed as part of #40, but it needs to be fixed.  If you can leave a TODO here for now, that would be enough.